internal/template/column.go

```diff
@@ -4,11 +4,12 @@ package template
 
 import (
 	"fmt"
-	"reflect"
 
 	"github.com/cheekybits/genny/generic"
```

Comment:
it seems we are still using the "old" `genny` from `cheekybits` here.
it's perhaps just confusing instead of wrong ?